packages/trpc/server/routers/viewer/organizations/listMembers.handler.ts

```diff
@@ -177,20 +177,29 @@ export const listMembersHandler = async ({ ctx, input }: GetOptions) => {
       let attributes;
 
       if (expand?.includes("attributes")) {
-        attributes = await prisma.attributeOption.findMany({
-          where: {
-            assignedUsers: {
-              some: {
-                memberId: membership.id,
-              },
+        attributes = await prisma.attributeToUser
```

Comment:
I flipped this query around so we dont need to do a nested query (sql generated is much cleaner)